plugins/hls-stan-plugin/src/Ide/Plugin/Stan.hs

```diff
@@ -72,15 +87,66 @@ rules :: Recorder (WithPriority Log) -> PluginId -> Rules ()
 rules recorder plId = do
   define (cmapWithPrio LogShake recorder) $
     \GetStanDiagnostics file -> do
-      config <- getPluginConfigAction plId
-      if pluginEnabledConfig plcDiagnosticsOn config then do
+      plugConfig <- getPluginConfigAction plId
+      if pluginEnabledConfig plcDiagnosticsOn plugConfig then do
           maybeHie <- getHieFile file
           case maybeHie of
             Nothing -> return ([], Nothing)
             Just hie -> do
-              let enabledInspections = HM.fromList [(LSP.fromNormalizedFilePath file, inspectionsIds)]
-              -- This should use Cabal config for extensions and Stan config for inspection preferences is the future
-              let analysis = runAnalysis Map.empty enabledInspections [] [hie]
+              let currentHSfromHIEAbs = hie_hs_file hie
+              currentHSfromHIERel <- liftIO $ makeRelativeToCurrentDirectory currentHSfromHIEAbs
+              -- This codes follows what 'runStan' does, from the module 'Stan'
+
+              -- There aren't any cli args. isLoud=False=Silent output
+              let isLoud = False -- Should this be enabled when debugging? Enables default stan cli output
```

Comment:
what happens if it's enabled? Does it write to stderr or what? we don't want that!